graylog2-web-interface/src/logic/content-packs/Entity.js

```diff
@@ -64,6 +64,18 @@ export default class Entity {
     return this.getValueFromData('description') || '';
   }
 
+  /* eslint-disable-next-line class-methods-use-this */
+  get isEntity() {
+    return true;
+  }
+
+  static [Symbol.hasInstance](obj) {
```

Comment:
Can we add a comment to this (and other occurrences) of this construct? I have no idea what it's doing. :smile:
Thank you!